mayastor/src/bdev/nexus/nexus_share.rs

```diff
@@ -33,49 +42,38 @@ impl Nexus {
             let name = format!("crypto-{}", self.name);
 
             // constant
-            let flavour = CString::new(CRYPTO_FLAVOUR)?;
+            let flavour = CString::new(CRYPTO_FLAVOUR).unwrap();
```

Comment:
This will crash the system, can we go back to "?" here?

Reply:
We discussed this offline. The summary: For certain basic failures like memory allocation failures or invalid strings that come from trusted sources, we can (or should) assert that the things are as we expect them as it makes the error handling easier. 👍